src/blog/2025/10/plc-to-mqtt-using-flowfuse.md

```diff
@@ -0,0 +1,187 @@
+---
+title: "How to Connect Any PLC to MQTT in Under an Hour"
+subtitle: "Connect any PLC to MQTT without the typical complexity, costs, and expertise requirements"
+description: "Learn how to extract data from Siemens, Allen-Bradley, Omron, Mitsubishi, OPC UA, and Modbus, transform it, and publish to MQTT using FlowFuse—without expensive gateways or consultants."
+date: 2025-10-24
+keywords: PLC to MQTT, Siemens S7 to MQTT, Allen-Bradley to MQTT, Omron to MQTT, Mitsubishi to MQTT, Modbus to MQTT, OPC UA to MQTT, PLC to MQTT, FlowFuse, Node-RED, industrial IoT
```

Comment:
```suggestion
keywords: MQTT, PLC, Siemens, S7, Allen-Bradley, EtherNet/IP, Omron, FINS, Mitsubishi, MC Protocol, Modbus, Modbus-RTU, Modbus-TCP, OPC-UA, FlowFuse, Node-RED, Industrial IoT, IIoT
```

Just a suggestion (for users searching by protocol name e.g. FINS or MC Protocol). Also, the format is more "keyword" centric rather than a bunch of "sentences" 

However, SEO is not my strong suit - you decide.